contrib/src/sceneeditor/collisionWindow.py

```diff
@@ -294,7 +294,7 @@ def createEntryField(self, parent, catagory, id, value,
         widget.configure(entry_state = initialState)
         widget.pack(side=LEFT)
         self.widgetDict[catagory+id] = widget
-        if defaultButton and (defaultFunction!=None):
+        if defaultButton and (defaultFunctionis notNone):
```

Comment:
It looks like you just replaced == with is , and != with is None, however that fails because most of the time there are no spaces, how did you replace those files, and did you have a look at them before committing them?